packages/core/src/embeddings/OpenAIEmbedding.ts

```diff
@@ -30,7 +44,8 @@ export class OpenAIEmbedding extends BaseEmbedding {
   constructor(init?: Partial<OpenAIEmbedding> & { azure?: AzureOpenAIConfig }) {
     super();
 
-    this.model = init?.model ?? OpenAIEmbeddingModelType.TEXT_EMBED_ADA_002;
+    this.model = init?.model ?? "text-embedding-ada-002";
+    this.dimensions = init?.dimensions;
```

Comment:
backward compatibility’s would be good to have a default dimensions as well

Reply:
Good point. So the way this works is the dimensions is undefined if the user doesn't specify so it automatically uses the default dimensions of the model (by not sending the parameter) but agreed this is not obvious just from the code. Will add some comments to make this clearer.

Reply:
https://github.com/run-llama/LlamaIndexTS/pull/445/commits/f27fd93b881c6ecbdc257d1286724f0faf8ee0c4 Really appreciate the feedback Emanuel!